### PR TITLE
Fix jointlock proc, better timers on joint and dislocating, joint deals less pain damage

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request

joint time doesnt scale off ROB anymore(yes, a ROB debuff!!!!!!!), dislocating and jointlock show progress bar because they are long
also fix
and blance
## Why It's Good For The Game
QoL love
fix good
yet another balance change nobody asked for
## Changelog
:cl:
fix: jointlock
tweak: progress bars for joint and dislocating
balance: less pain damage on joint
/:cl:

### CODE REVIEW

File: code/modules/mob/mob_grab_specials.dm [hunk 1]
```diff
@@ -47,19 +47,16 @@
...
 		return
-
 	var/obj/item/organ/external/organ = target.get_organ(check_zone(target_zone))
 	if(!organ || organ.dislocated == -1)
 		return
 
-	var/time_to_jointlock = max( 0, ( target.getarmor(target_zone, ARMOR_MELEE) - attacker.stats.getStat(STAT_ROB) ) )
-	if(!do_mob(attacker, target, time_to_jointlock))
-		attacker << SPAN_WARNING("You must stand still to jointlock [target]!")
+	if(!do_after(attacker, 70, target))
```
Comment (SirRichardFrancis) [inline code/modules/mob/mob_grab_specials.dm:54]: Flat seven seconds? That's quite long for a combat action.
```suggestion
	if(!do_after(attacker, 7 SECONDS, target))
```

**Review by SirRichardFrancis — COMMENTED** [review]
What have been fixed exactly and where are QoL improvements?
I see nothing but needless balance changes with awful description.

**Comment by Kegdo** [comment]
> What have been fixed exactly and where are QoL improvements? I see nothing but needless balance changes with awful description.

jointlocking wasn't working. at all.
QoL - progress bars on long actions, which means you know when they end, duh

File: code/modules/mob/mob_grab_specials.dm (hunk 1 above)
Comment (Kegdo) [reply code/modules/mob/mob_grab_specials.dm:54]: lotsa pain, intended, cool and good

**Comment by Kegdo** [comment]
> Orly? Then how would you explain that?
> 
> image

you dumb fr*icking fucking dumbass, this is dislocating, THERE IS JOINT LOCKING, GRAB INTENT DISARM LEVEL

> So you changed action to take a long time, slapped progress bar on it and calling that QoL?

yes

**Comment by Kegdo** [comment]
image

CLEARLY
image
You didn't read the code, nor proc, nor play the game

**Comment by SirRichardFrancis** [comment]
> this is dislocating, THERE IS JOINT LOCKING, GRAB INTENT AGGRESSIVE LEVEL

Whatever. Tho message is actually missing.
image

> you dumb fr*icking fucking dumbass

Seethe.

> image

Cope.

> CLEARLY You didn't read the code nor play the game

Dilate.

**Comment by Kegdo** [comment]
if you don't get an announcement message, the feature automatically does not work

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**